hail/python/hail/methods/qc.py

```diff
@@ -976,7 +978,12 @@ def __str__(self):
         builder.append('Allele type distribution')
         builder.append('------------------------')
         for allele_type, count in Counter(self.allele_types).most_common():
-            builder.append(f'  {allele_type_formatter % allele_type}: {count} alternate alleles')
+            summary = f'  {allele_type_formatter % allele_type}: {count} alternate alleles'
+            if allele_type == 'SNP':
+                nti = self.nti
+                ntv = self.ntv
```

Comment:
Why rebind these as `nti` and `ntv` instead of just directly using `self.nti` in the fstring?

Reply:
seems about equivalent, looked sorta gross with the division though